Gemfile.lock

```diff
@@ -266,125 +268,127 @@ GEM
-      aws-sigv4 (~> 1.0)
-    aws-sdk-kms (1.23.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-kms (1.77.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.45.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-s3 (1.143.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sdk-kms (~> 1)
-      aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.6.1)
+      aws-sigv4 (~> 1.8)
+    aws-sigv4 (1.8.0)
       aws-eventstream (~> 1, >= 1.0.2)
-    base64 (0.1.1)
+    base64 (0.2.0)
+    bigdecimal (3.1.6)
     binding_of_caller (0.8.0)
       debug_inspector (>= 0.0.1)
-    bootsnap (1.4.5)
-      msgpack (~> 1.0)
+    bootsnap (1.18.3)
+      msgpack (~> 1.2)
     bootstrap-sass (3.4.1)
       autoprefixer-rails (>= 5.2.1)
       sassc (>= 2.0.0)
     bootstrap3-datetimepicker-rails (4.17.47)
       momentjs-rails (>= 2.8.1)
     brakeman (5.4.1)
     builder (3.2.4)
-    bundler-audit (0.9.0.1)
+    bundler-audit (0.9.1)
       bundler (>= 1.2.0, < 3)
       thor (~> 1.0)
     byebug (8.2.5)
-    chef-utils (18.1.29)
+    chef-utils (18.4.2)
       concurrent-ruby
-    coderay (1.1.1)
+    coderay (1.1.3)
     commonmarker (0.23.10)
-    concurrent-ruby (1.2.2)
-    connection_pool (2.2.1)
-    crack (0.4.3)
-      safe_yaml (~> 1.0.0)
+    concurrent-ruby (1.2.3)
+    connection_pool (2.4.1)
+    crack (0.4.6)
+      bigdecimal
+      rexml
     crass (1.0.6)
-    dalli (3.2.3)
+    dalli (3.2.7)
+      base64
     ddtrace (0.45.0)
       msgpack
     debug_inspector (0.0.3)
     diffy (3.4.2)
-    dogstatsd-ruby (3.0.0)
-    domain_name (0.5.20190701)
-      unf (>= 0.0.5, < 1.0.0)
-    doorkeeper (5.6.6)
+    dogstatsd-ruby (3.3.0)
+    domain_name (0.6.20240107)
+    doorkeeper (5.6.8)
       railties (>= 5)
-    dotenv (2.2.1)
+    dotenv (2.8.1)
     encryptor (3.0.0)
     erubi (1.12.0)
-    erubis (2.7.0)
-    execjs (2.7.0)
-    faraday (2.7.11)
+    execjs (2.9.1)
+    faraday (2.8.1)
       base64
       faraday-net_http (>= 2.0, < 3.1)
       ruby2_keywords (>= 0.0.4)
-    faraday-http-cache (2.5.0)
+    faraday-http-cache (2.5.1)
       faraday (>= 0.8)
     faraday-multipart (1.0.4)
       multipart-post (~> 2)
     faraday-net_http (3.0.2)
     faraday-net_http_persistent (2.1.0)
       faraday (~> 2.5)
       net-http-persistent (~> 4.0)
-    ffi (1.15.5)
+    ffi (1.16.3)
     ffi-compiler (1.0.1)
       ffi (>= 1.0.0)
       rake
-    flay (2.12.0)
-      erubis (~> 2.7.0)
+    flay (2.13.2)
+      erubi (~> 1.10)
       path_expander (~> 1.0)
       ruby_parser (~> 3.0)
       sexp_processor (~> 4.0)
     flowdock (0.7.1)
       httparty (~> 0.7)
       multi_json
-    forking_test_runner (1.2.0)
+    forking_test_runner (1.13.0)
       parallel_tests (>= 1.3.7)
-    git_diff_parser (3.1.0)
-    gitlab (4.10.0)
-      httparty (~> 0.14, >= 0.14.0)
-      terminal-table (~> 1.5, >= 1.5.1)
-    globalid (1.1.0)
-      activesupport (>= 5.0)
+    git_diff_parser (3.2.0)
+    gitlab (4.19.0)
```

Comment:
### [gitlab](https://github.com/NARKOZ/gitlab)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 4.10.0 → 4.19.0

[_[change-log](https://github.com/NARKOZ/gitlab/releases), [source-code](https://github.com/NARKOZ/gitlab), [gem-diff](https://my.diffend.io/gems/gitlab/4.10.0/4.19.0)_]





<details>
<summary>Commits</summary>

A change of **241** commits. See the full changes on [the compare page](https://github.com/NARKOZ/gitlab/compare/v4.10.0...v4.19.0).

These are the first 10 commits:
- (0397776) [Container Registry API](https://github.com/NARKOZ/gitlab/commit/0397776d60fd4ca326e9619c3f84846fdc58006e)
- (64cabc4) [Merge pull request #485 from akkee/471-container_registry_AP…](https://github.com/NARKOZ/gitlab/commit/64cabc47ced8b598ad4e1ed3b8653315a0a28811)
- (02aa5c0) [Markdown API](https://github.com/NARKOZ/gitlab/commit/02aa5c09e7e01d2d2c9586de86ada166171e127e)
- (bd472d8) [Doc and argument fixes in markdown API](https://github.com/NARKOZ/gitlab/commit/bd472d8a79bdd4ac154ac66a29fecb1b76cc414c)
- (c934d31) [Merge pull request #486 from akkee/472-markdown_API](https://github.com/NARKOZ/gitlab/commit/c934d31b4602d4e363f36a4d1099092b53b1dc2c)
- (9c6abb9) [Valid .gitlab-ci.yml API](https://github.com/NARKOZ/gitlab/commit/9c6abb911ef218baa68338c65b446b700bfb5f98)
- (7e5dbc7) [Update stale.yml](https://github.com/NARKOZ/gitlab/commit/7e5dbc7c9d2bb527aed40f54bc7c36d3635ae42f)
- (7f563c4) [Merge branch 'master' into 473-validate_gitlab_ci_yml_API](https://github.com/NARKOZ/gitlab/commit/7f563c46d951b40c5484e3278afbf691d73941ed)
- (979f1ef) [Merge pull request #487 from akkee/473-validate_gitlab_ci_ym…](https://github.com/NARKOZ/gitlab/commit/979f1ef4e7310d91f30d66f6ca96223eadbca5d6)
- (573e7b0) [Added missing endpoints in Runners API](https://github.com/NARKOZ/gitlab/commit/573e7b054b8abc69800223ecdc0531dfc8a77ed0)


</details>